src/bin/travis/afterSuccess.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "BUILD_TYPE='$BUILD_TYPE'"
+echo "TRAVIS_BRANCH='$TRAVIS_BRANCH'"
+echo "TRAVIS_PULL_REQUEST='$TRAVIS_PULL_REQUEST'"
+
+if [ "$BUILD_TYPE" == "sbt" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+
+    if [ "$TRAVIS_BRANCH" == "develop" ]; then
+        sbt 'set test in Test := {}' publish
+
+    elif [[ "$TRAVIS_BRANCH" =~ ^[0-9\.]+_hotfix$ ]]; then
```

Comment:
Discussed offline. Will only be pushing hotfix branches to docker, for now. Thus to kick off this process, the current operating procedure should be to create a new hotfix git branch whenever we release.